client/sites-dashboard/components/sites-table-row.tsx

```diff
@@ -166,7 +167,9 @@ export default function SitesTableRow( { site }: SiteTableRowProps ) {
 						/>
 					</Column>
 					<Column mobileHidden>{ site.plan.product_name_short }</Column>
-					<Column mobileHidden>July 16, 1969</Column>
+					<Column mobileHidden>
+						{ site.options?.updated_at ? <TimeSince date={ site.options.updated_at } /> : '-' }
```

Comment:
Are you sure we want to display `-` instead of simply hiding the column? In which cases will we not get the timestamp?

Reply:
The only case I found is when I add `updated_at` to the store, the old version is still stored in local storage, and the store is not refreshed yet. I'm good with leaving it empty as such case too.

@p-jackson, as you proposed using the dash, what do you think about leaving it empty? It seems that leaving that empty might be less disruptive.

Reply:
Empty sounds good too. In don't think `updated_at` will be undefined very often, so I just wanted to do whatever was easiest to satisfy type safety and not crash at runtime (so not removing the column sounded harder and not worth it)

Reply:
Thanks. I've changed it to empty then, it even seems to be cleaner.